secure/key-manager/src/main.rs

```diff
@@ -51,6 +51,10 @@ fn main() {
 }
 
 fn create_and_execute_key_manager(key_manager_config: KeyManagerConfig) -> Result<(), Error> {
+    info!(LogSchema::new(LogEntry::Initialized)
+        .event(LogEvent::Pending)
+        .key_manager_config(&key_manager_config));
```

Comment:
I'm of the opinion to not stick it in the Schema struct since you're using it exactly once, but @bmwill might have a different opinion.  You could just do a  

```
info!(
  LogSchema(...).event(...),
  key_manager_config = ?key_manager_config
);
```

Reply:
Yeah, don't feel super strongly here either. I'll follow the pattern of avoiding a new schema struct because it requires less code.. Seems this is what LSR and LibraNode do anyway. 😄